examples/lines_bars_and_markers/step_demo.py

```diff
@@ -29,6 +29,15 @@
 plt.legend(title='Parameter where:')
 plt.show()
 
+# The following example demonstrates the drawstyle parameter that can be used 
```

Comment:
Please add a line of '#' to begin a new [sphinx gallery section](https://sphinx-gallery.github.io/syntax.html#embed-rst-in-your-example-python-files) and make this render as html, not as comment in the code. Compare also the references block bleow.